examples/workspace/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "devDependencies": {
+      "@playwright/test": "1.50.1"
```

Comment:
Should we use `1.51.0` since this needs to match the `playwright_version = "1.51.0",` in the `playwright_repository(` and the browsers

Reply:
Yeah I think your follow up is a good resolution